frontend/src/layout/lemonade/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -24,11 +24,12 @@
 .Breadcrumbs__breadcrumb {
     display: flex;
     align-items: center;
-    color: var(--primary);
-    font-weight: 500;
+    color: $text_default;
+    font-weight: $medium;
```

Comment:
Total nit: IMO, different than colors for instance, there isn't much benefit to using a variable for font-weight, since there are only 4 options and 500 will just stay 500 (e.g. we won't ever change it to 520 because it's not a thing, and if we change it to 600 than we'll need to switch to different var, negating the point).

Reply:
The rationale to have `$medium` as a variable is to avoid using other weights, we only use normal (400), medium (500) and bold (600), we don't use 100, 700, ..., when using pure numbers it can be hard to forget that. Aside from better consistency, it'll enable us to support custom fonts if we decide to go down that route.